README.md

```diff
@@ -14,6 +14,8 @@ This website uses [jekyll](https://jekyllrb.com/) to build static HTML. Those ar
     
 This will compile the markdown code and start a server at [http://localhost:4000/](http://localhost:4000/) which you can then open in your browser.
 
+If you get errors when building native extensions with bundle, make sure you have the necessary libraries. Specifically zlib is needed.
```

Comment:
Shouldn't you rather add the missing package to the `apt install` line?

Reply:
Good idea!